research/object_detection/box_coders/faster_rcnn_box_coder.py

```diff
@@ -107,8 +107,8 @@ def _decode(self, rel_codes, anchors):
       tx /= self._scale_factors[1]
       th /= self._scale_factors[2]
       tw /= self._scale_factors[3]
-    w = tf.exp(tw) * wa
-    h = tf.exp(th) * ha
+    w = tf.exp(tf.minimum(tw, 80)) * wa
```

Comment:
Add a comment explaining how you get the number 80 here?

Reply:
Avoid INF in exp and w,h below. We set 80 as a upper_bound since tf.exp(89) yields INF.